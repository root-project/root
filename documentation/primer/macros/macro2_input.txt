# Measurement of Friday 26 March
# Experiment 2 Physics Lab

1   6   5
2   12  5
3   14  4.7
4   20  4.5
5   22  4.2
6   24  5.1
7   35  2.9
8   45  4.1
9   44  4.8
10  53  5.43