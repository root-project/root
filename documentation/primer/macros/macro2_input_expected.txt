# Measurement of Friday 26 March
# Experiment 2 Physics Lab
# Expected points from theory predictions

1   6   0.5
2   12  1.
3   18  1.5
4   24  2.0
5   30  3.7
6   36  4.9
7   42  5.4
8   48  6.8
9   54  7.5
10  60  9.7